kimchi/cairo/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+pub mod runner;
```

Comment:
Thanks for the suggestion! I will be adding it :)

In this case I am calling it the cairo runner as this is the name they use in their paper. It basically refers to the piece of code that executes (runs) the cairo program and obtains the memory instance that will be used for the proof generation. So it is supposed to contain the instruction-related functions, and the "state transition logic" as you say, but from an execution perspective, rather than constraints-checking. Maybe I could state this point clearer, as I have changed the modules names very often.